docs/index.rst

```diff
@@ -48,6 +49,24 @@ The configuration parameters that can be set on this page are;
 
   - **Min Reporting Interval**: This control the minimum interval between reports of data changes in subscriptions. It sets an upper limit to the rate that data will be ingested into the plugin and is expressed in milliseconds.
 
+  - **Asset Naming Scheme**: The plugin can ingest data into a number of different assets based on the selection of the asset naming scheme
+
+    +-----------+
+    | |opcua_5| |
+    +-----------+
+
+     - *Single datapoint* An asset will be created for each variable read from the OPC/UA server. The asset will contain a single datapoint whose name will be taken from the browse name of the variable read. The asset name will be created by appending the browse name of the variable to the fixed asset name prefix defined in the *Asset Name* configuration option above.
+
+     - *Single datapoint object prefix* An asset will be created for each variable read from the OPC/UA server. The asset will contain a single datapoint whose name will be taken from the browse name of the variable read. The asset name will be created by appending the browse name of the variable to the browse name of the variables parent object.
+
+     - *Asset per object* An asset will be created for each OPC/UA object that is subscribed to. That asset will be named using the browse name of the OPC/UA object and will contain a datapoint per variable within the OPC/UA object. The name of the datapoint will be the browse name of the variable.
+
+     - *Single asset* A single asset will be created with all the variables read from the OPC/UA server as datapoints within that asset. The asset name will be taken from the *Asset Name* configuration item and the datapoint name from the browse name of the OPC/UA variable.
+
+     When an asset is created that has multiple datapoints within it, all datapoints may not be included in any single reading as the plugin only gets data that has changed since the last reading has been taken.
+
+     The plugin uses the browse names of the OPC/UA variables to name the datapoints with an asset, there is no requirement for the browse names to be unique, the plugin resolves this by detecting duplicates browse names and if one is found appending the node ID to the browse name to create a unique datapoint name. This is important for both *Single datapoint* and *Single Asset* modes as in the first case the asset name is the same as the datapoint name and is global for all OPC/UA variables that are monitored. The result of this would be that the same asset name and datapoint name would be used to store multiple OPC/UA variables. In the case of *Single Asset* mode the datapoints names are kept within a single asset and hence an error would occur if two datapoints had the same name.
```

Comment:
Suggest breaking up this first sentence into multiple sentences. How about: 'Since the plugin uses the browse names of the OPC/UA variables to name the datapoints with an asset, there is no requirement for the browse names to be unique. The plugin resolves this by detecting duplicate browse names. If one is found, the node ID is appended to the browse name to create a unique datapoint name."